frontend/src/components/Edit.js

```diff
@@ -24,7 +24,7 @@ export default function Edit({ id, reloadList, cancelHandler, editService }) {
 
   return (
     <Wrapper onSubmit={nameSubmitHandler}>
-      <InputField placeholder="new name" onChange={nameHandler} />
+      <InputField placeholder="new" onChange={nameHandler} />
```

Comment:
you want to make the input a controlled component, add `value={newName.newValue}`

And the think about the names of your state, having a state called name with an object with a newValue key is a bit confusing and probably to complicated if your new name is only a string ;)